app/src/main/java/com/android/sample/ui/utils/ConfirmationPopUp.kt

```diff
@@ -33,32 +32,30 @@ fun ConfirmationPopUp(
     onConfirm: () -> Unit,
     onDismiss: () -> Unit,
     titleText: String,
-    confirmationText: String,
+    confirmationText: String = "",
```

Comment:
Is this the descriptive text under the title in the alert dialog? Would it be possible to give this a more descriptive name? Ex: bodyText?

Reply:
Done !